src/app/modules/auth-private/login/login.component.ts

```diff
@@ -65,6 +65,7 @@ export class LoginComponent implements OnInit {
   }
 
   onLoggedIn() {
+    this.tokenService.checkTokenExpiration();
     this.tokenService.tokenHandler();
```

Comment:
If `checkTokenExpiration` has to be done every time before `tokenHandler` then this can be handled inside `tokenHandler` (or another helper function) itself

Reply:
Done. 

Since we wanted to remove the GraphQL error in priority we made these changes. For now, no new access token and expiry date are created/updated when the user logs in to the app. 
In future scope: Will be adding the functionality of updating those when the user logs in to the app.

Added the same comment in the ticket.